core/lib/include/log.h

```diff
@@ -51,6 +51,9 @@ extern "C" {
  */
 enum {
     LOG_NONE,       /**< Lowest log level, will output nothing */
+    LOG_WTF,        /**< What a Terrible Failure log level, will print
```

Comment:
```suggestion
    LOG_WTF = 0401,        /**< What a Terrible Failure log level, will print
```

Ensure that everyone gets it!